irc.html

```diff
@@ -10,24 +10,28 @@ <h1>Channels</h1>
 <div>
   <p>
     Join one of our <abbr title="Internet Relay Chat">IRC</abbr> channels on Freenode:<br/>
+    To chat on #minetest-hub you need to ask an operator for voice. You can't use the webchat on #minetest-dev.<br>
   </p>
   <ul>
     <li>
       <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel for general Minetest engine and games discussion or
```

Comment:
Almost suggests modding can't be discussed here.
`Main channel for general Minetest engine, modding and games discussion or support.`